admin/view/template/extension/mobbex/finance_widget.twig

```diff
@@ -0,0 +1,112 @@
+{{ header }}{{ column_left }}
+<div id="content">
+    <div class="page-header">
+        <div class="container-fluid">
+            <div class="pull-right">
+                <button type="submit" form="mobbex_conf" data-toggle="tooltip" title="save" class="btn btn-primary"><i class="fa fa-save"></i></button>
+                <a href="{{ cancel }}" data-toggle="tooltip" title="cancel" class="btn btn-default"><i class="fa fa-reply"></i></a>
+            </div>
+            <h1>{{ heading_title }}</h1>
+            <ul class="breadcrumb">
+                {% for breadcrumb in breadcrumbs %}
+                    <li><a href="{{ breadcrumb.href }}">{{ breadcrumb.text }}</a></li>
+                {% endfor %}
+            </ul>
+        </div>
+    </div>
+
+    <div class="container-fluid">
+        {% if error_warning %}
+            <div class="alert alert-danger"><i class="fa fa-exclamation-circle"></i> {{ error_warning }}
+                <button type="button" class="close" data-dismiss="alert">&times;</button>
+            </div>
+        {% endif %}
+    
+        <div class="panel panel-default">
+            <div class="panel-heading">
+                <h3 class="panel-title"><i class="fa fa-pencil"></i>Edit Mobbex</h3>
+            </div>
+            <div class="panel-body">
+                <form action="{{ action }}" method="post" enctype="multipart/form-data" id="mobbex_conf" class="form-horizontal">
+
+                    {# Payment Method Status Field #}
+                    <div class="form-group required">
+                        <label class="col-sm-2 control-label" for="input-status">{{ status_label }}</label>
+                        <div class="col-sm-10">
+                            <select name="module_mobbex_finance_widget_status_widget" id="input-status" class="form-control">
+                            {% if module_mobbex_finance_widget_status_widget %}
+                                <option value="1" selected="selected">{{ text_enabled }}</option>
+                                <option value="0">{{ text_disabled }}</option>
+                            {% else %}
+                                <option value="1">{{ text_enabled }}</option>
+                                <option value="0" selected="selected">{{ text_disabled }}</option>
+                            {% endif %}
+                            </select>
+                        </div>
+                    </div>
+
+                    {# Active on Product #}
+                    <div class="form-group required">
+                        <label class="col-sm-2 control-label" for="input-status">{{ active_product_label }}</label>
+                        <div class="col-sm-10">
+                            <select name="module_mobbex_finance_widget_active_product" id="input-status" class="form-control">
```

Comment:
El atributo `id` debería ser distinto en cada opción para que al cliquear el label te seleccione la opción correcta (también modificar atributo `for` de cada label).